DietPi-Software | RPi-Monitor: Fix CPU temperature meter

Joulinar opened the pull request:
There is an issue to display CPU temp correctly on RPI-Monitor. It's basically an old issue and RPI-Monitor has not been updated for long. But still we should fix it. 😄 

**Status**: Ready
- [x] adjust `dietpi-siftware`
- [x] tested on RPI4B 64bit
- [x] tested on RPI3B+ 32bit

**Reference**: 
- https://dietpi.com/phpbb/viewtopic.php?t=10001
- https://github.com/XavierBerger/RPi-Monitor/issues/374

**Commit list/description**:
- DietPi-Software | RPI-Monitor - fix an issue to display CPU Temperature correctly

Joulinar committed:
v8.1

DietPi-Software | RPI-Monitor - fix an issue to display CPU Temperature correctly

MichaIng commented on dietpi/dietpi-software:
```diff
@@ -8795,6 +8795,9 @@ _EOF_
 			G_EXEC /usr/share/rpimonitor/scripts/updatePackagesStatus.pl
 
+			# Fix issue to display CPU Temperature correctly https://github.com/XavierBerger/RPi-Monitor/issues/374
+			G_CONFIG_INJECT 'dynamic.1.postprocess=' 'dynamic.1.postprocess=$1/1000' /etc/rpimonitor/template/temperature.conf
```
To mute shellcheck, since obviously the `$1` is meant literally and not as positional parameter to expand, you can add the following line above `G_CONFIG_INJECT`:
```sh
# shellcheck disable=SC2016
```
Such makes shellcheck skip this rule for the next command/statement.

Joulinar replied:
will do, juts playing with it. Still need to find out how to round the value to two digits 🤔

Joulinar committed:
v8.1

DietPi-Software | RPI-Monitor - change on how temperature is displayed

Joulinar committed:
v8.1

DietPi-Software | RPI-Monitor - satisfy shellcheck

MichaIng committed:
v8.1

- DietPi-Software | RPi-Monitor: Show up to two decimal numbers with fixed CPU temperature
- DietPi-Software | RPi-Monitor: Do not add USB HDD gauge when it is the rootfs, since it is then covered by "SD card" already (misleading but all info is there). Support non-ext4 USB HDDs, be a little more strict when parsing "df" and replace postprocess with "df -m" flag

MichaIng committed:
v8.1

- CHANGELOG | RPi-Monitor: Resolved an issue where the CPU temperature was not shown. Many thanks to @KamikazeePL for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=10001

Joulinar committed:
v8.1

DietPi-Software | RPI-Monitor - fix typo

MichaIng commented on dietpi/dietpi-software:
```diff
@@ -8821,7 +8821,7 @@ dynamic.14.rrd=GAUGE
 web.status.1.content.9.name=USB HDD
 web.status.1.content.9.icon=usb_hdd.png
-web.status.1.content.9.line.1="<b>/sda1</b> Used: <b>"+KMG(data.usbhdd_used,'M')+"</b> (<b>"+Percent(data.udbhdd_used,data.usbhdd_total,'M')+"</b>) Free: <b>"+KMG(data.usbhdd_total-data.usbhdd_used,'M')+ "</b> Total: <b>"+ KMG(data.usbhdd_total,'M') +"</b>"
+web.status.1.content.9.line.1="<b>/sda1</b> Used: <b>"+KMG(data.usbhdd_used,'M')+"</b> (<b>"+Percent(data.usbhdd_used,data.usbhdd_total,'M')+"</b>) Free: <b>"+KMG(data.usbhdd_total-data.usbhdd_used,'M')+ "</b> Total: <b>"+ KMG(data.usbhdd_total,'M') +"</b>"
```
Good find 👍.

Joulinar replied:
was not me. I just copied it 😜

MichaIng replied:
Ah, then to be forwarded 🙂.
